### ISSUE TITLE
Commit button icon not refreshing when working folderis  on a  linux based synology box

### ISSUE THREAD

**Issue by starryvistas** [gitextensions/gitextensions#2835]
We have our developer working folders and base repo  on a remote share and run VS2012 and Git-extensions from our windows 7 pc's. I know its not recommended to have the working folder on a share  but this worked fine when the share was on a windows 2008 server. Now we have moved those working folders and the bare repo to the synology box the commit button icon does not reflect the correct commit status eg it remains red after a commit but clicking it pulls up a dialog showing there is nothing to commit

**Comment by vbjay** [comment]
A blocking git process is running.

**Comment by vbjay** [comment]
This is why you you should use remotes. You can have a hook script run when
a commit is pushed and check out that commit or branch automatically. Look
up git deploy.

**Comment by starryvistas** [comment]
Thanks for the reply. I separated the central and personal repositories onto two different Synology boxes. I  moved central repo onto Synology box 1 and cloned it to a personal repo on Synolgy box 2  and ran  Git-Extensions off my Windows 7 pc. Commits to the personal repo were still not registered properly in the Commit button icon. The repos were accessed by Git-extensions via shared folders created from the synology desktop/dasbooard.  If  a personal repo is created on a remote windows Server 2008 by cloning from the Synology's central repo, then commits to that personal repo thru git-extensions are displayed correctly. This is the solution we have settled on.
Though it looks to me that, as far as git-extensions is concerned, a linux shared folder mapped as a Windows drive is not behaving exactly like a folder hosted on a Windows machine mapped windows share.
